catalog/opensearch/catalog-opensearch-api/pom.xml

```diff
@@ -16,54 +16,27 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-
     <parent>
-        <artifactId>catalog-opensearch-pom</artifactId>
+        <artifactId>opensearch</artifactId>
```

Comment:
✏️ Total nit, but can we make the ordering of groupid : artifactid : version consistent?